cloud_init.tf

```diff
@@ -4,17 +4,24 @@ data "template_cloudinit_config" "cloud_init_tasks" {
   gzip          = true
   base64_encode = true
 
+  # Note: The filename parameters in each part below are only used to name the
+  # mime-parts of the user-data.  It does not affect the final name for the templates.
+  # For the x-shellscript parts, it will also be used as a filename in the scripts
+  # directory.
+
   part {
-    filename     = "openvpn-config"
+    filename     = "openvpn-config.yml"
     content_type = "text/cloud-config"
     content = templatefile(
       "${path.module}/cloudinit/openvpn-config.tpl.yml", {
         private_networks = var.private_networks
         client_network   = var.client_network
     })
+    merge_type = "list(append)+dict(recurse_array)+str()"
```

Comment:
Are the two `merge_type` lines necessary?  I don't remember the innards of cloud-init wizardry right now.

If they are required, they probably warrant a comment.

Reply:
I also seem to recall us only needing one `merge_type` line in the past for a similar situation.

Reply:
@felddy, after you reminded me I am onboard with the `merge_type`.  I'd like to see comments there, though, so I don't need a personal refresher class in cloud-init next time.

Reply:
See: 
https://github.com/search?q=org%3Acisagov+merge_type&type=Code

Reply:
I added 3e613e9 for you @jsf9k .  Good idea.